iron-image.html

```diff
@@ -105,9 +114,11 @@
 
   <template>
 
-    <img id="img" role="none" hidden$="[[_computeImageVisibility(sizing)]]">
-    <div id="placeholder" hidden$="[[_computePlaceholderVisibility(fade,loaded,preload)]]" class$="[[_computePlaceholderClassName(fade,loaded,preload)]]"></div>
-    <content></content>
+    <div id="sizedImgDiv" role="img" aria-label$="{{alt}}" hidden$="[[_computeBackdropHidden(sizing)]]"></div>
+    <img id="img" alt$="{{alt}}" hidden$="[[_computeImgHidden(sizing)]]">
```

Comment:
This looks really nice.

However, one thing I hadn't considered is that this means the default case will be that this will be
```
<img alt="" src=...>
```
This means that unless the user _specifies_ an `alt` attribute, the image will be hidden from screen readers (won't show up at all, as if it's not there to begin with). 
Is there some way to set the default value to be `null` and judiciously add this attribute only if the `alt` attribute on the host is non-null?

Similarly, if the user _does_ set `alt=""` on the host, `#sizedImagDiv` should probably be `aria-hidden`, to get the same effect.

(For extra credit, it would be great to set a placeholder `aria-label` on `#sizedImgDiv` in that case. For `<img>` elements, screen readers do the job on their own if no `alt` attribute is provided, but in this case it won't have enough information to do that since this isn't an `<img>`. The usual fallback is the filename part of the `src`.)